usr/share/rear/verify/VEEAM/default/400_verify_veeam.sh

```diff
@@ -13,6 +13,9 @@ fi
 Log "Deleting Veeam SQLite Linux Agent Database"
 rm -rf /var/lib/veeam/*
 
+LogPrint "Mounting /tmp with tmpfs"
+mount tmpfs /tmp -t tmpfs -o mode=1777,strictatime,nosuid,nodev || Error "Failed to mount /tmp with tmpfs"
```

Comment:
I think that this won't work on a 2nd attempt to run `rear recover` which is something quite common